recipe/migrations/cuda102_110_111_112_115_116_ppc64le.yaml

```diff
@@ -0,0 +1,68 @@
+migrator_ts: 1645016609
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: true
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+
+c_compiler_version:            # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+cxx_compiler_version:          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+fortran_compiler_version:      # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+  - 9                          # [ppc64le]
+
+cuda_compiler_version:         # [ppc64le]
+  - 10.2                       # [ppc64le]
+  - 11.0                       # [ppc64le]
+  - 11.1                       # [ppc64le]
+  - 11.2                       # [ppc64le]
+  - 11.5                       # [ppc64le]
+  - 11.6                       # [ppc64le]
```

Comment:
We build with only up to 11.2 and use those for 11.5 for x86

Reply:
@isuruf 
I want to build CUDA 21.12.00 for ppc64le which requires CUDA 11.5+as per requirements listed in the link - https://github.com/rapidsai/cudf/blob/v21.12.00/CONTRIBUTING.md#setting-up-your-build-environment.
So, I am not sure if building on 11.2 and using those for 11.5 will work. I have tried building CUDF 21.12.00 on CUDA 11.2 locally and it fails due to missing header files/references. 
Can you please let me know if we can add atleast one of those preferably CUDA 11.6?

Reply:
How would you do build cudf for x86_64? It should be the same way.

Reply:
I have not built cudf on x86_64. But when I install cudatoolkit 11.2 on ppc64le and try to build cudf 21.12.00 then there reference errors.

Reply:
I can confirm that cudf requires building with CUDA 11.5 or newer and then can run on 11.0 or newer. It uses new compiler features / support.

Reply:
That's irrelevant. You can use cuda 11.2 compiled dependencies with cudatoolkit 11.6 when you want to build cudf.

Reply:
@isuruf 
Ok, removed the 11.5 and 11.6 cuda versions. Can you please approve this PR now?

Reply:
No. Please see my other comment.

Reply:
@isuruf ,
Can you please help in the other comment and let me know how that can be achieved?